tools/gulp/docs/generateDocPage.js

```diff
@@ -0,0 +1,72 @@
+const Docs = require('../../../packages/docs/src/scripts/Docs').default;
+const React = require('react');
+const ReactDOMServer = require('react-dom/server');
+const savePage = require('./savePage');
+
+/**
+ * Create an HTML page with the documentation's UI
+ * @return {Promise}
+ */
+function generateDocPage(routes, page, rootPath) {
+  const componentRenderer = () => {
+    if (process.env.NODE_ENV === 'development') {
+      // In development mode we let the client handle all of the React rendering,
+      // since if we were generating the HTML pages in our build process, Gulp would
+      // need restarted each time a React file changes, which is super annoying.
+      return '';
+    }
+
+    return ReactDOMServer.renderToString(<Docs page={page} routes={[]} />);
+  };
+
+  if (rootPath) {
+    rootPath = `${rootPath}/`;
+  }
+
+  const head = `${seo(page, rootPath)}
+  <link rel="shortcut icon" type="image/x-icon" href="/${
+    rootPath
```

Comment:
This is unfortunate formatting 😞

Reply:
There's a second occurrence of this later too

Reply:
ugh yea..Prettier is doing this